Cargo.toml

```diff
@@ -52,9 +57,10 @@ std = ["digest/std", "pkcs1/std", "pkcs8/std", "rand_core/std", "signature/std"]
 pem = ["pkcs1/pem", "pkcs8/pem"]
 pkcs5 = ["pkcs8/encryption"]
 getrandom = ["rand_core/getrandom"]
+algoid = ["spki/alloc", "const-oid"]
```

Comment:
I know it's nauseatingly verbose but I think I'd probably prefer `algorithm-identifier` or failing that `algorithm-id` for this just as there aren't otherwise canonical abbreviations